flutter/lib/mobile/pages/remote_page.dart

```diff
@@ -531,7 +531,13 @@ class _RemotePageState extends State<RemotePage> {
                       controller: _textController,
                       // trick way to make backspace work always
                       keyboardType: TextInputType.multiline,
-                      onChanged: handleSoftKeyboardInput,
+                      // onChanged: handleSoftKeyboardInput,
+                      onChanged: (newValue) async {
+                        await Future.delayed(const Duration(milliseconds: 10));
```

Comment:
since only ios has this problem, please enable this for ios only. No need to introduce extra code for working env.

Reply:
Also add comment here, who can remember why modify this after several months?

Reply:
Fixed and append some comments.